docs/Hotkeys.md

```diff
@@ -0,0 +1,31 @@
+## Introduction to Custom Hotkeys
+
+By default Chatterino contains a set of default _hotkeys_. A hotkey can be described using five parameters:
+
+- Its name
+- A [category](#categories)
+- An [action](#actions)
+- A key combo
+- The [arguments](#arguments)
+
+![Open quick switcher hotkey being edited](images/hotkeys/Editing.png)
+
+### Categories
+
+Categories describe where in the app the hotkey's action take place. For example:
```

Comment:
```suggestion
Categories describe where in the app the hotkeys action takes place. For example:
```
It stuck out since it was the only `hotkey's` vs all the other `hotkeys`